tests/unit/trajectory/plotting.test.ts

```
@@ -0,0 +1,268 @@
+      step,
+      metadata: props,
+    })),
+  }
+}
+
+// Simple test data extractor
+function test_extractor(frame: TrajectoryFrame): Record<string, number> {
+  const data: Record<string, number> = { Step: frame.step }
+  if (frame.metadata) {
+    for (const [key, value] of Object.entries(frame.metadata)) {
+      if (typeof value === `number`) data[key] = value
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)